Source/Core/Core/BootManager.cpp

```diff
@@ -392,6 +392,15 @@ bool BootCore(const std::string& _rFilename)
   SConfig::GetInstance().m_SYSCONF->SetData("IPL.PGS", StartUp.bProgressive);
   SConfig::GetInstance().m_SYSCONF->SetData("IPL.E60", StartUp.bPAL60);
 
+  if (StartUp.bWii)
+  {
+    // Disable WiiConnect24's standby mode. If it is enabled, it prevents us from receiving
+    // shutdown commands in the State Transition Manager (STM).
+    // TODO: remove this if and once Dolphin supports WC24 standby mode.
+    SConfig::GetInstance().m_SYSCONF->SetData("IPL.IDL", 0x00);
```

Comment:
This commit caused memory corruption in the sysconf code because `0x00` is promoted to `int` when resolving the template type for `SetData`. This overwrites off the end of the "IPL.IDL" entry, which is a SmallArray of size 1.  
I'm working on rewriting SysConf code which will make it nicer and should make it harder for such bugs to happen.